pubspec.yaml

```diff
@@ -21,11 +21,9 @@ environment:
   sdk: ">=2.7.0 <3.0.0"
 
 dependencies:
-  cached_network_image: ^2.2.0+1
+  built_collection: ^4.3.2
+  built_value: ^7.1.0
   http: ^0.12.2
```

Comment:
Please try to use something more powerful than http, Chopper for example, or dio

Reply:
Replaced http with chopper